pkg/controller/node/util/controller_utils.go

```diff
@@ -77,6 +77,9 @@ func DeletePods(kubeClient clientset.Interface, recorder record.EventRecorder, n
 			continue
 		}
 
+		// Set pod phase to Unknown, since it may caused by node unresponsive
+		pod.Status.Phase = v1.PodUnknown
```

Comment:
We have never set or used the `Unknown` phase before.  I'm concerned that adding this here is going to break things, even if we check for it in a few places. 

I think from an API perspective it's too late to add this - it's effectively adding a new phase to the pod enumeration because no one has ever had to deal with it.  @kubernetes/api-reviewers